index.html

```diff
@@ -9,6 +9,68 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1> 
+  <div class="content">
+    <h1 class="title">Front-End Tools</h1>
+    <div class="grid-flex-display"> 
+      <div class="tech-card">
+        <img class="tech-card-image" src="assets/react_icon.svg">
+        <h2 class="tech-card-title">ReactJs</h2>
+        <span class="tech-card-release">Release: 2013</span>
+        <p class="tech-card-description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+      </div>
+      <div class="tech-card">
+        <img class="tech-card-image" src="assets/angular_icon.svg"/>
+        <h2 class="tech-card-title">AngularJs</h2>
+        <span class="tech-card-release">Release: 2010</span>
+        <p class="tech-card-description">AngularJS (commonly referred to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architechture.</p>
+      </div>
+      <div class="tech-card">
+        <img class="tech-card-image" src="assets/vue_icon.svg"/>
+        <h2 class="tech-card-title">VueJs</h2>
+        <span class="tech-card-release">Release: 2014</span>
+        <p class="tech-card-description">VueJS (pronounced /vjuː/, like view) is an open-source progressive JavaScript framework for building user interfaces.</p>
+      </div>
+      <div class="tech-card">
+        <img class="tech-card-image" src="assets/elm_icon.svg"/>
+        <h2 class="tech-card-title">Elm</h2>
+        <span class="tech-card-release">Release: 2016</span>
+        <p class="tech-card-description">
+          Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="tech-card">
+        <img class="tech-card-image" src="assets/polymer_icon.svg"/>
+        <h2 class="tech-card-title">Polymer</h2>
+        <span class="tech-card-release">Release: 2015</span>
+        <p class="tech-card-description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+    </div>
+      <h1 class="title display-layout-title-margin">Back-End Tools</h1>
+      <div class="grid-flex-display horizontal-center"> 
```

Comment:
the other wrapper doesn't have the `horizontal-center` class, I think it's not necessary